frost-link.md

```diff
@@ -5,36 +5,78 @@
 ## API
 
 | Attribute | Type | Value | Description |
-| --------- | ---- | ----- | ----------- |
-| `disabled` | `boolean` | `false` | **default**: Click to redirect from one route to another |
-| | | `true` | :no_entry_sign: Can't click this! :notes: |
-| `size` | `string` | `small` | The smallest link you ever did see |
-| | | `medium` | **default**: Not quite as small as `small`, but not very big either |
-| | | `large` | Now *that's* what I call a link! |
-| `priority` | `string` | `primary` | Call-to-action :telephone: |
-| | | `secondary` | **default**: Run of the mill, garden variety  |
-| | | `tertiary` | Low-key, subdued  |
+| --------- | ---- | ----- | ----------- 
+| `priority` | `string` | `primary` | primary link - opens content in a new tab |
+|  |  | `secondary` | secondary link - opens content in the same tab |
+|  |  | `tertiary` | tertiary link - opens content in the same tab |
+|  |  | `action` | action link - opens content in the same tab |
+| `size` | `string` | `small` | small size link |
+|  |  | `medium` | medium size link |
+|  |  | `large` | large size link |
+| `design` | `string` | `info-bar-action` | custom styling for applications that use links but don't follow the button styling.  Requires `icon` to be specified.  Should not be used with `priority` and `size`. |
```

Comment:
Can we make the description clearer?  e.g. 'link styling for the info-bar context'